### DISCUSSION TITLE
Notifiche di Trascrizioni e Annotazioni automatiche per lo Scioglimento delle Unioni Civili.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#916] [Miglioramenti] [answered]
Buongiorno,

segnaliamo una ridondanza nelle Notifiche che vengono prodotte, successivamente alla validazione/firma degli eventi generati per uno scioglimento dell'Unione Civile.

L'iter si realizza, tramite i 3 eventi successivi, di: 'Manifestazione congiunta di volontà di scioglimento dell'unione civile tra persone dello stesso sesso', 'Accordo di scioglimento degli uniti civilmente davanti all’Ufficiale di Stato Civile', e la 'Conferma dell'accordo di scioglimento dell'unione civile tra persone dello stesso sesso'.
Per ognuno di questi eventi, vengono generate 2 Notifiche di Trascrizione (una per ogni parte dell'Unione).
In considerazione del fatto che solo la conferma dell’accordo ha una conseguenza anagrafica, riteniamo che dovrebbero essere generate, solo le Notifiche di Trascrizione, a fronte della conferma dell’accordo, e non per gli altri 2 eventi.

Oltre alle Notifiche di Trascrizione, per gli stessi eventi, vengono generate le seguenti Notifiche di Annotazioni automatiche:
per l'Accordo di scioglimento, una per l'Unione Civile, e due per le Nascita (una per ogni parte dell’Unione);
anche per la Conferma dell'Accordo, una per l'Unione Civile, e due per le Nascite (una per ogni parte dell’Unione).

In considerazione del fatto che solo la conferma dell’accordo ha una conseguenza sull’atto di nascita, riteniamo che le notifiche di annotazione sull’atto di nascita, dovrebbero essere generate, solo alla conferma dell’accordo. Inoltre, dal momento che sull’atto di Unione Civile deve essere annotata sia la manifestazione di volontà allo scioglimento, sia lo scioglimento, riteniamo che dovrebbero essere prodotte due notifiche per l'Unione, una alla firma dell’evento di manifestazione di volontà (adesso non prodotta), ed una alla firma dell’evento di conferma dello scioglimento dell’unione civile. Mentre non dovrebbe essere prodotta la notifica per l'Unione, alla firma dell’evento di accordo di scioglimento.

In attesa di riscontro,

cordiali saluti.

**Comment by mttfranci** [comment, marked as answer]
Salve, la segnalazione è in lavorazione in relazione anche alla issue #677. Sarà presumibilmente inclusa nella prossima minor version.

Saluti 

Assistenza ANSC
